src/ProxyManager/ProxyGenerator/Util/PublicScopeSimulator.php

```diff
@@ -108,8 +108,8 @@ private static function getUndefinedPropertyNotice(string $operationType, string
             . '            \'Undefined property: %s::$%s in %s on line %s\',' . "\n"
             . '            get_parent_class($this),' . "\n"
             . '            $' . $nameParameter . ',' . "\n"
-            . '            $backtrace[0][\'file\'],' . "\n"
-            . '            $backtrace[0][\'line\']' . "\n"
+            . '            $backtrace[0][\'file\'] ?? \'\',' . "\n"
```

Comment:
In which scenarios would this be the case? Can you add a functional test (`.phpt`)?